packages/core/src/components/menu/_common.scss

```diff
@@ -45,6 +45,15 @@ $dark-menu-item-color-active: $dark-minimal-button-background-color-active !defa
     color: $pt-text-color-disabled;
   }
 
+  &.pt-menu-item-single-line {
```

Comment:
😢 sad we need an extra class just for that

Reply:
Alternatively we could define a class for multiline instead. Would that be preferable?